Marsiple/Common/Network/MartianApiManager.swift

```diff
@@ -22,20 +21,28 @@ class DataFetcher {
                             .appendingPathComponent("\(id)")
                             .appendingPathComponent(MartianApi.URLs.comments)
             else { return DispatchQueue.main.async { failure(DataFetcherError.urlCreationFailure) } }
-        getData(url: url, method: HTTPRequestMethod.get, success: success, failure: failure)
+        getData(url: url, success: success, failure: failure)
+    }
+    
+    static func postComment(comment: Comment) {
+        guard let url = MartianApi.url?
+                            .appendingPathComponent(MartianApi.URLs.posts)
+                            .appendingPathComponent("\(comment.postId)")
+                            .appendingPathComponent(MartianApi.URLs.comments)
+            else { return }
+        let json: [String: String] = ["name": comment.name, "email": comment.email, "body": comment.body]
+        do {
+            let jsonData = try JSONSerialization.data(withJSONObject: json, options: .prettyPrinted)
+            postData(data: jsonData, url: url)
+        } catch {
+            print("Error serializing Json")
+        }
```

Comment:
This whole `JSON` related block should be ejected. Like I mentioned previously, we `will not` work with any `JSON` frameworks outside of `Codable`. 

There is a solution to ejecting the `id`, but still using `Codable` 🙂 Can help you with that one if you're stuck.